pkg/commands/compute/language_go.go

```diff
@@ -125,14 +159,92 @@ func (g *Go) Build() error {
 	return bt.Build()
 }
 
+// identifyTinyGoConstraint checks the compute-sdk-go version used by the
+// project and if it's less than 0.2.0 we'll change the TinyGo constraint to be
+// version 0.26.0
+//
+// We do this because the 0.2.0 release of the compute-sdk-go bumps the TinyGo
+// version requirement to 0.28.1 and we want to avoid any scenarios where a
+// bump in SDK version causes the user's build to break (which would happen for
+// users with a pre-existing project who happen to update their CLI version: the
+// new CLI version would have a TinyGo constraint that would be higher than
+// before and would stop their build from working).
+//
+// NOTE: The `configConstraint` is the latest CLI application config version.
+// If there are any errors trying to parse the go.mod we'll default to the
+// config constraint.
+func identifyTinyGoConstraint(configConstraint, fallback string) string {
```

Comment:
If this turns out to be too fragile, we can switch to https://pkg.go.dev/golang.org/x/mod/modfile

Reply:
Cool thanks. I think our use case is simple enough but yeah if in the real-world it's starts being a problem we can pick up a tool like that 👍🏻